src/profile.c

```diff
@@ -242,6 +253,7 @@ void profile_set_home(bool state) {
     if (state) {
         led_static_mask(LED_ALL);
         led_set_mode(LED_MODE_ENGAGE);
+        hold_home_to_sleep_timer = time_us_64();  // Restart countdown.
```

Comment:
Such functionality should not be part of the profile handling. Even for the home profile.